src/DynamoUtilities/DynamoFeatureFlagsManager.cs

```diff
@@ -0,0 +1,117 @@
+﻿using Dynamo.Utilities;
+using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Diagnostics;
+using System.IO;
+using System.Reflection;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace DynamoUtilities
+{
+
+
+    /// <summary>
+    /// A wrapper around the DynamoFeatureFlags CLI tool.
+    /// Which is itself a wrapper around LaunchDarkly.
+    /// Not thread safe.
+    /// </summary>
+    internal class DynamoFeatureFlagsManager : CLIWrapper
+    {
+        private string relativePath = Path.Combine("DynamoFeatureFlags", "DynamoFeatureFlags.exe");
+        private Dictionary<string, object> AllFlagsCache { get; set; }//TODO lock is likely overkill.
+        private SynchronizationContext syncContext;
+        internal static event Action FlagsRetrieved;
+        public override void Dispose()
```

Comment:
Does this need to override the base class Dispose? The base Dispose is killing the process anyway.

Reply:
good spot, can remove this.